### PR TITLE
Bootstrapping

### PR DESCRIPTION
**Pull request by seemantk**
added some bootstrap components for aesthetic rearrangement of the legend and controls.
@curran, do you have time to review/merge?

note, that the buttons don't show/hide yet.

### CODE REVIEW

**Comment by curran** [comment]
Woohoo! Will review now.

I started working on adding the note toggle checkbox over at https://github.com/sul-cidr/josquin-ribbon/pull/77 . Will try to merge this into that.

File: index.html [hunk 1]
```diff
@@ -19,23 +20,42 @@
     <h2>Work in Progress</h2>
   </header>
   <article>
-    <div class="container">
+    <div class="container-fluid">
```
Comment (curran) [inline index.html:23]: Nice.

File: index.html [hunk 2]
```diff
@@ -19,23 +20,42 @@
...
+            <div id="notes"></div>
+            <div id="nav"></div>
+          </div><!--.row-->
         </div><!--.notes-->
-        <div class="col-xs-12 col-lg-2">
-          <div id="combine-separate-ui"></div>
-          <div id="extreme-notes-ui"></div>
-          <div id="ribbons-ui"></div>
-          <div id="legend"></div>
+        <div class="col-md-3 col-lg-3">
```
Comment (curran) [inline index.html:31]: This is sweet! Love the Boootstrappyness.

File: js/baton.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ var width = 960
   , notesNav = NotesNav()
       .svg(d3.select("#nav").append("svg"))
   , notesBook = NotesBook()
-      .svg(d3.select("#notes").append("svg"))
+      .svg(d3.select("#notes"))
```
Comment (curran) [inline js/baton.js:7]: Nice change.

File: js/combineSeparateUI.js [hunk 4]
```diff
@@ -10,27 +10,34 @@ function CombineSeparateUI(){
...
+                .attr("aria-label", function(d) { return d; })
           , inputs = labels
               .append("input")
                 .attr("type", "radio")
                 .attr("name", "combine-separate-group")
                 .property("checked", function(d, i) { return !i; })
           , spans = labels
               .append("span")
-                .text(function (d){ return " " + d + " "; })
+                .html(function (d){ return d; })
```
Comment (curran) [inline js/combineSeparateUI.js:32]: Interesting, I didn't know of this method.

File: js/ribbonsUI.js [hunk 5]
```diff
@@ -8,12 +8,12 @@ function RibbonsUI(){
       , ribbonMode = Ribbon.STANDARD_DEVIATION
       , modes = [
           {
-            label: "Standard Deviation",
-            mode: Ribbon.STANDARD_DEVIATION
+            label: "Standard Deviation"
+            , mode: Ribbon.STANDARD_DEVIATION
```
Comment (curran) [inline js/ribbonsUI.js:12]: Sorry about that, I'll try to stick to this style.

**Review by curran — APPROVED** [review]
Great work @seemantk !

**Comment by curran** [comment]
Works great locally. Will merge.

**Comment by curran** [comment]
Thanks @davidmcclure, whoever you are, for managing our labels :+1:

**Comment by seemantk** [comment]
Thanks for your kind words, @curran.  I've just pushed an update that shows/hides the ribbon buttons (same branch)

**Comment by curran** [comment]
Awesome, thank you @seemantk . Merged over at https://github.com/sul-cidr/josquin-ribbon/pull/78

**Comment by davidmcclure** [comment]
Hi @curran and @seemantk,

Uh, I have no idea why it says I removed that label, I certainly didn't do it on purpose. (Nor was I anywhere close to a computer an hour ago.) Freaky, I'll try to figure out what's up.

In the meantime - sorry about this, I certainly have no strong feelings about whether this is in progress or not!

**Comment by curran** [comment]
@davidmcclure Haha that's funny, you've been adding and removing labels ever since we started this project. Maybe you're attached to a bot of some kind?

**Comment by davidmcclure** [comment]
@curran Wow, not good, really sorry about this. I think this is related to https://waffle.io/, a Trello-like wrapper around GH issues, though I haven't used it in months. Looking at the account - it looks like repos in the sul-cidr account are getting automatically linked into my profile, since I'm a part of that GH group, and then those `in progress` tags are getting added automatically when issues are created, or something?

Will fix, apologies.

**Comment by curran** [comment]
No problem at all. It's actually been pretty cool having the labels updated automatically. We don't mind it at all :) Just thought it was a bit odd.

Good sleuthing!

**Comment by curran** [comment]
@davidmcclure No problem at all. It's actually been pretty cool having the labels updated automatically. We don't mind it at all :) Just thought it was a bit odd.

Good sleuthing!

**Pull request merged**